code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -104,6 +104,14 @@
 	amount_per_transfer_from_this = 20
 	list_reagents = list("ephedrine" = 10, "coffee" = 10)
 
+/obj/item/weapon/reagent_containers/hypospray/medipen/leppen //basilisks were buffed

```

Comment:
So are half the other ones, but got make sure its written meme like "i ded pls nerf basilisks"